src/Tasks/CreateItem.cs

```diff
@@ -128,14 +134,64 @@ private List<ITaskItem> CreateOutputItems(Dictionary<string, string> metadataTab
             return outputItems;
         }
 
+        /// <summary>
+        /// Attempts to expand wildcards and logs warnings or errors for attempted drive enumeration.
+        /// </summary>
+        private bool TryExpandingWildcards(ITaskItem[] expand, bool includeAttribute)
+        {
+            const string CreateItemTask = "CreateItem";
+            string attributeType;
+            string fileSpec;
+
+            FileMatcher.SearchAction searchAction;
+
+            if (includeAttribute)
+            {
+                (Include, searchAction, fileSpec) = ExpandWildcards(expand);
+                attributeType = XMakeAttributes.include;
```

Comment:
nit: Can you pass `attributeType` as a parameter of this method instead of `includeAttribute` and eliminate the `if`/`else`?